Source/Cli/cli.cpp

```diff
@@ -68,6 +68,10 @@ int Cli::exec(QCoreApplication &a)
         {
             showVersion = true;
         }
+        else if(a.arguments().at(i) == "--resetsettings")
+        {
+            Preferences().resetSettings();
+        }
```

Comment:
About https://github.com/bavc/qctools/pull/696#issuecomment-737999853, I added a flag for that, so just need to add something like 

```
            std::cout << "Settings cleared." << std::endl;
            configIsSet = true;
```